CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - [[#267](https://github.com/rust-vmm/kvm-ioctls/pull/267)]: Added `HypercallExit` field to `VcpuExit::Hypercall` and added `ExitHypercall` to `Cap`.
+- [[#270](https://github.com/rust-vmm/kvm-ioctls/pull/267)]: Propagate `MemoryFault` exit reason in `KVM_RUN`.
```

Comment:
There's an associated capability, `KVM_CAP_MEMORY_FAULT_INFO`, for this, let's include that in this PR too

Reply:
done